packages/pass-style/src/types.d.ts

```diff
@@ -1,6 +1,32 @@
 /* eslint-disable no-use-before-define */
 import { PASS_STYLE } from './passStyle-helpers.js';
 
+/**
+ * Tagged has own `{ [PASS_STYLE]: "tagged", [Symbol.toStringTag]: $tag }`.
+ *
+ * Remotable has a prototype chain in which the penultimate object has own
+ * `{ [PASS_STYLE]: "remotable", [Symbol.toStringTag]: $iface }`, where both
+ * $tag and $iface must be strings, and the latter must either
+ * be "Remotable" or start with "Alleged: " or "DebugName: ".
+ *
+ * Selector has its own
+ * `{ [PASS_STYLE]: "selector", [Symbol.toStringTag]: description }`.
+ */
+export type PassStyleMarker = 'tagged' | 'remotable' | 'symbol';
```

Comment:
Not for this PR

Reply:
But do keep the refactoring rename from `TaggedOrRemotable` to `PassStyleMarker`

Reply:
Done in #2793